lib/commands/recordscreen.js

```diff
@@ -0,0 +1,129 @@
+import _ from 'lodash';
+import { retryInterval } from 'asyncbox';
+
+let commands = {}, extensions = {};
+
+const RETRY_PAUSE = 1000;
+
+/**
+* record the display of devices running Android 4.4 (API level 19) and higher.
+* It records screen activity to an MPEG-4 file. Audio is not recorded with the video file.
+*
+* @param filePath the video file name
+*                 for example, "/sdcard/demo.mp4"
+* @param videoSize the format is widthxheight.
+*                  if it is "default", the default value is the device's native display resolution (if supported),
+*                  1280x720 if not. For best results,
+*                  use a size supported by your device's Advanced Video Coding (AVC) encoder.
+*                  for example, "1280x720"
+* @param timeLimit the maximum recording time, in seconds. if it is -1, the default and maximum value is 180 (3 minutes).
+* @param bitRate the video bit rate for the video, in megabits per second.
+*                if it is -1, the default value is 4Mbps. You can increase the bit rate to improve video quality,
+*                but doing so results in larger movie files.
+*                for example, 6000000
+*
+*/
+commands.startRecordingScreen = async function ( filePath, videoSize, timeLimit, bitRate ) {
+  let cmd, data, length, indexNoSuchFile;
```

Comment:
this method seems to be failing when it's called from the script, no idea why. I can dig a bit more over the weekend. 
btw; the time limit is 3minutes, we may need to set this on the documentation. 
https://developer.android.com/studio/command-line/adb.html#screenrecord

I will also set defaults for the extra args 
`async function(filePath, videoSize = 'default', timeLimit = 180, bitRate = -1) {`